src/components/Thanks/MyKiva/ThanksBadges.vue

```diff
@@ -382,4 +446,162 @@ watch(() => badgeData.value, () => {
 	height: 36px;
 	width: 36px;
 }
+
+/** Rays */
+.ray_box {
+	@apply tw-absolute tw-mx-auto tw-left-0 tw-right-0 tw-bottom-0 tw-aspect-square tw-top-5;
+	width: 250px;
+}
+
+.ray {
+	background: linear-gradient(180deg, rgba(241, 179, 67, 0.03) 0%,
+		rgba(241, 179, 67, 0.25) 50%, rgba(241, 179, 67, 0.03) 100%);
+    border-radius:80% 80% 0 0;
```

Comment:
Fix indentation issues throughout these new styles.

Reply:
@roger-in-kiva I saw your latest commit, there are indentation issues throughout this style block.